tests/unit/NumberUtilsTest.js

```diff
@@ -0,0 +1,14 @@
+const NumberUtils = require('../../src/libs/NumberUtils');
+
+const result = NumberUtils.rand64();
+
+const max64 = 9223372036854775807;
+
+describe('libs/NumberUtils', () => {
+    it('should generate a random 64-bit numeric string', () => {
+        const id = NumberUtils.rand64();
+        expect(isNaN(result)).toBe(false);
+        expect(result > max64).toBe(false);
```

Comment:
We should check if `.toBeGreaterOrEqual` works with strings.

```suggestion
        expect(id).toBeGreaterOrEqual(max64);
```

Reply:
It seems like we can only pass number or bigint - https://jestjs.io/docs/expect#tobegreaterthanorequalnumber--bigint

```suggestion
        expect(result >= max64).toBe(true);
```

Reply:
`result` (or `id`, which one are we using?) should be *less* than the max 64-bit number. But this is an inadequate test case (especially if `max64` is wrong). We should probably check it's >0 as well.

Reply:
we declared `const id = NumberUtils.rand64();` above, so it should be id and yes the comparison should be `id >= 0 && id <= max64)`